kafka-ui-react-app/package.json

```diff
@@ -35,13 +35,27 @@
     "typesafe-actions": "^5.1.0",
     "typescript": "~3.7.4"
   },
+  "lint-staged": {
+    "*.{js,ts,jsx,tsx}": [
+      "eslint -c .eslintrc.json --fix",
+      "git add"
+    ],
+    "*.js": "eslint --cache --fix"
+  },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
+    "lint": "esprint check",
+    "lint:fix": "esprint check --fix",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "mock": "node ./mock/index.js"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
```

Comment:
```suggestion
      "pre-commit": "yarn tsc --noEmit && lint-staged"
```

Reply:
For preventing of commit uncompilable ts code.

Reply:
my bad, when I was reinstalling husky looks like it overwrote the pre-commit hook